### PR TITLE
Add r-coop

### PR DESCRIPTION
**Pull request by mtekman**

### CODE REVIEW

File: recipes/r-coop/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    based on the input data, handled by R''s S3 methods.  Full descriptions of the algorithms
+    and benchmarks are available in the package vignettes.'
+
+  license_family: BSD
+
+extra:
+  recipe-maintainers:
+  - mtekman
+
+# The original CRAN metadata for this package was:
```
Comment (bgruening) [inline recipes/r-coop/meta.yaml]: please remove the comments

File: recipes/r-coop/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  summary: 'Fast implementations of the co-operations: covariance, correlation, and cosine similarity.  The
+    implementations are fast and memory-efficient and their use is resolved automatically
+    based on the input data, handled by R''s S3 methods.  Full descriptions of the algorithms
+    and benchmarks are available in the package vignettes.'
+
+  license_family: BSD
+
+extra:
+  recipe-maintainers:
+  - mtekman
```
Comment (bgruening) [inline recipes/r-coop/meta.yaml]: have you used the conda_skeleton_helper script? You can add a few more maintainers here.

File: recipes/r-coop/meta.yaml (hunk 1 above)
Comment (mtekman) [reply recipes/r-coop/meta.yaml]: what to do about the toolchain?

File: recipes/r-coop/meta.yaml (hunk 2 above)
Comment (mtekman) [reply recipes/r-coop/meta.yaml]: Is it okay if I add you?

Comment (bgruening) [reply recipes/r-coop/meta.yaml]: https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml those can be added.

**Pull request merged**